ffs/api/api_actions.go

```diff
@@ -0,0 +1,150 @@
+package api
```

Comment:
Splitted `api.go` in some more logical files, so we can find things easily and don't get so long files.